static/app/components/feedback/feedbackItem/feedbackScreenshot.tsx

```diff
@@ -0,0 +1,169 @@
+              onClick={onPrevious}
+              icon={<IconChevron direction="left" />}
+              size="xs"
+            />
+            {tct('[currentScreenshot] of [totalScreenshots]', {
+              currentScreenshot: screenshotInFocus + 1,
+              totalScreenshots,
+            })}
+            <Button
+              disabled={screenshotInFocus + 1 === totalScreenshots}
+              aria-label={t('Next Screenshot')}
+              onClick={onNext}
+              icon={<IconChevron direction="right" />}
+              size="xs"
+            />
+          </StyledPanelHeader>
+        )}
+        <StyledPanelBody hasHeader={totalScreenshots > 1}>
+          {loadingImage && (
+            <StyledLoadingIndicator>
+              <LoadingIndicator mini />
+            </StyledLoadingIndicator>
+          )}
+          <StyledImageWrapper
+            onClick={() => openVisualizationModal(screenshotAttachment)}
+          >
+            <StyledImageVisualization
+              attachment={screenshotAttachment}
+              orgId={orgSlug}
+              projectSlug={projectSlug}
+              eventId={eventId}
+              onLoad={() => setLoadingImage(false)}
+              onError={() => setLoadingImage(false)}
+            />
+          </StyledImageWrapper>
+        </StyledPanelBody>
+      </Fragment>
+    );
+  }
+
+  return (
+    <Role organization={organization} role={organization.attachmentsRole}>
+      {({hasRole}) => {
+        if (!hasRole) {
+          return null;
+        }
+        return <StyledPanel>{renderContent(screenshot)}</StyledPanel>;
+      }}
+    </Role>
+  );
+}
+
+export default FeedbackScreenshot;
+
+const StyledPanel = styled(Panel)`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 0;
+  max-width: 360px;
+  max-height: 360px;
+  border: 0;
+`;
+
+const StyledPanelHeader = styled(PanelHeader)`
+  padding: ${space(1)};
+  width: 100%;
+  border: 1px solid ${p => p.theme.border};
+  border-bottom: 0;
+  border-top-left-radius: ${p => p.theme.borderRadius};
+  border-top-right-radius: ${p => p.theme.borderRadius};
+  display: flex;
+  justify-content: space-between;
+  text-transform: none;
+  background: ${p => p.theme.background};
+`;
+
+const StyledPanelBody = styled(PanelBody)<{hasHeader: boolean}>`
+  border: 1px solid ${p => p.theme.border};
+  min-height: 48px;
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  flex: 1;
+
+  ${p =>
+    !p.hasHeader &&
+    `
+  border-top-left-radius: ${p.theme.borderRadius};
+  border-top-right-radius: ${p.theme.borderRadius};
+  `}
+`;
+
+const StyledLoadingIndicator = styled('div')`
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+`;
+
+const StyledImageWrapper = styled('div')`
+  :hover {
+    cursor: zoom-in;
+  }
+`;
```

Comment:
Lets make this extend a `styled('button')` so we get some keyboard accessibility for free from the browser, it'll require a few css properties to be overridden/disabled.

Also, we can set the cursor to `zoom-in` at all times, when the mouse isn't hovered no one can see it anyway.

```suggestion
const StyledImageWrapper = styled('div')`
  :hover {
    cursor: zoom-in;
  }
`;
```